.github/labeler.yml

```diff
@@ -0,0 +1,78 @@
+# Based on paths provided in the CODEOWNERS file
+
+# Add 'awaiting-review' to any PR opened (starting with just team-ExternalDeps)
+awaiting-review:
+- src/main/java/com/google/devtools/build/lib/bazel/bzlmod/**/*
+- src/test/java/com/google/devtools/build/lib/bazel/bzlmod/**/*
+- src/test/py/bazel/bzlmod/**/*
+- src/main/java/com/google/devtools/build/lib/bazel/repository/**/*
+- src/test/java/com/google/devtools/build/lib/bazel/repository/**/*
+# - '**/*'
```

Comment:
Remove `# - '**/*'` ?

Reply:
I left it in there because we'll need it in the next phase. Is that alright?
We'll have to remove the external deps directories from here so that the label is added to all opened PRs.